src/configurations/destinations/http/ui-config.json

```diff
@@ -457,6 +457,22 @@
           {
             "name": "Request Body",
             "fields": [
+              {
```

Comment:
@krishna2020 We can address this issue without explicitly requesting a root key. Instead, we can implement basic validation on the mappings, ensuring that they all begin with the same prefix key.

Reply:
Yes. But from the UX perspective, for every field user has to mention the same prefix. Also, it won't be implicit for the user to understand that common prefix has to be maintained - they have to understand the validation error thrown & then fix it. Instead, with the separate root key field, we are making it more explicit to the user on XML format requirements

Reply:
Also, we want to support product behaviour of delivering the events just with Initial setup configuration. In that case, without root key (and default value of `root`), XML format configuration can't be supported if the event payload has more than one root level keys

Reply:
@sandeepdsvs we need to add proper notes in UI to explain the root key will be common prefix for all the field mappings being defined in the mapping component

Reply:
if we have preview things will be much clearer so we need to prioritize that next